.ci/test/static/run

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eo pipefail
+
+#
+# This script runs static tests which
+# can be performed without code execution.
+#
+# Some example are code sniffing and unit testing
+#
+
+# Run composer install to get the vendor directory
+composer install --no-ansi --no-interaction --optimize-autoloader --no-progress
+
+# Delete and reset the web directory
+# This is necessary so that third-party
+# dependencies are not tested
+rm -rf web
+git checkout -- web
```

Comment:
Instead of running `composer install` and then cleaning the web directory, could we just add the linter and code-sniff binaries to our docker container?

I guess this way allows the client to be in control of the version of the linter used (e.g. if using a newer version of PHP, or if there is a bugfix in the linter). Maybe that's rare?

If you prefer to stick with this technique, I would recommend replacing the rm / checkout with:
```
git clean -ffdX web
```

Reply:
Updated in `c2aeff7e22c7a35690a661dbce1d5f1ac33487b3`. I used `git clean -fdX web` as I think the double `f` was a typo.

Reply:
This should probably be ported to `example-wordpress-composer` as well